.github/workflows/jepsen-cluster-txm.yml

```diff
@@ -0,0 +1,69 @@
+name: jepsen-cluster-txm
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - master
+  repository_dispatch:
+    types: [backend_automation]
+  workflow_dispatch:
+
+env:
+  CI_MAKE: make -f .travis.mk
+
+jobs:
+  jepsen-cluster-txm:
+    # We want to run on external PRs, but not on our own internal PRs
+    # as they'll be run by the push to the branch.
+    if: ( github.event_name == 'push' ||
+        github.event_name == 'repository_dispatch' ||
+        github.event_name == 'workflow_dispatch' ||
+        github.event.pull_request.head.repo.full_name != github.repository ) &&
+        ! endsWith(github.ref, '-notest')
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+    # image built by .gitlab.mk instructions and targets from .travis.mk
+    container:
+      image: docker.io/tarantool/testing:debian-stretch
+      ports:
+        # Opens tcp port 2222 on the host for internal SSH port
+        - 2222:22
+      # Our testing expects that the init process (PID 1) will
+      # reap orphan processes. At least the following test leans
+      # on it: app-tap/gh-4983-tnt-e-assert-false-hangs.test.lua.
```

Comment:
Do we need this in jepsen jobs?

Reply:
Right, by default containers running in Github Actions start without ports opened externally. So SSH connection using 22 port is not possible without it, check [documentation](https://docs.github.com/en/actions/guides/about-service-containers#communicating-with-service-containers):
```
By default, all containers that are part of the same Docker network expose all ports to each other, and no ports are exposed outside of the Docker network.
```
```
When a job runs directly on a runner machine, the service running in the Docker container does not expose its ports to the job on the runner by default. You need to map ports on the service container to the Docker host. For more information, see "Mapping Docker host and service container ports."
```
[Mapping Docker host and service container ports.](https://docs.github.com/en/actions/guides/about-service-containers#mapping-docker-host-and-service-container-ports)